aospy/test/test_roundtrip.py

```diff
@@ -150,33 +151,53 @@ def test_vert_int_sigma(self):
                                  dtype_out_time='av',
                                  dtype_out_vert='vert_int',
                                  dtype_in_vert='sigma',
-                                 **self.temp_test_params)
+                                 **self.three_d_test_params)
+        calc = Calc(calc_int)
+        calc.compute()
+
+    def test_sub_daily(self):
+        """Not tested for now in models other than idealized_moist.
+        and idealized_moist_rad."""
+        if not getattr(self, 'sub_daily_test_params', False):
```

Comment:
That was my initial thought too, but the simplest way I could think of to check if we wanted to run the sub-daily test for a given model was to see if we defined the ```sub_daily_test_params``` attribute in the given class.  This requires referencing ```self```, which unfortunately I can't reference outside the method body.

Do you think there is a better way to do this using ```unittest.skipif``` instead?  This perhaps could also be a place where re-thinking the structure could help...but like you say given our commitments at the moment, not a huge priority.